js/okex.js

```diff
@@ -790,69 +790,67 @@ module.exports = class okex extends Exchange {
                 symbol = symbol + '-' + ymd;
             }
             if (option) {
-                const strikePrice = this.safeString (market, 'stk');
-                const optionType = this.safeString (market, 'optType');
+                strikePrice = this.safeString (market, 'stk');
+                optionType = this.safeString (market, 'optType');
                 symbol = symbol + '-' + strikePrice + '-' + optionType;
+                optionType = (optionType === 'P') ? 'put' : 'call';
             }
         }
         const tickSize = this.safeString (market, 'tickSz');
-        const precision = {
-            'amount': this.safeNumber (market, 'lotSz'),
-            'price': this.parseNumber (tickSize),
-        };
         const minAmountString = this.safeString (market, 'minSz');
         const minAmount = this.parseNumber (minAmountString);
-        let minCost = undefined;
-        if ((minAmount !== undefined) && (tickSize !== undefined)) {
-            minCost = this.parseNumber (Precise.stringMul (tickSize, minAmountString));
-        }
-        const active = true;
+        const minCost = this.parseNumber (Precise.stringMul (tickSize, minAmountString));
```

Comment:
I would completely remove this line, if the exchange does not define a real minimal cost, we should not derive it from precision.